README.md

```diff
@@ -102,7 +103,7 @@ _**Version where the locale was released. 4.x denotes `Unreleased`._
 [si]: https://github.com/npwsamarasinghe
 [sk]: https://github.com/xbotkaj
 [sr]: https://github.com/nebojsajsimic
-[sv-SE]: https://github.com/tangix
+[ta]: https://github.com/darthleslin
```

Comment:
don't delete sv-SE, just insert your own

Reply:
Changes DONE.
Thank You.